ooniprobe/Test/Test/WebConnectivity.mm

```diff
@@ -44,15 +45,17 @@ -(void)onEntry:(JsonResult*)json obj:(Measurement*)measurement{
 }
 
 -(int)getRuntime{
-    if (self.settings.options.max_runtime != nil){
+    if (self.settings.options.max_runtime != nil &&
+        ![self.settings.options.max_runtime isEqual: MAX_RUNTIME_DISABLED]){
```

Comment:
I believe you should check for the runtime being >= 0

Reply:
I prefer to use a constant (-1) to indicate the disabled. The user can't set lower than 10.